src/applications/simple-forms/21-0972/pages/claimantIdentification.js

```diff
@@ -15,7 +15,7 @@ export default {
         labels: claimantIdentificationOptions,
       },
       'ui:errorMessages': {
-        enum: 'Please select who you will be signing for',
+        required: 'Please select who you will be signing for',
```

Comment:
sometimes you might need both here. one that triggers related to the enum, and one that relates to the required. can you double check that only required is needed here?